idd/Energy+.idd.in

```diff
@@ -57223,17 +57223,41 @@ Chiller:Electric:ReformulatedEIR,
        \note curve = a + b*CWS + c*CWS**2 + d*LCT + e*LCT**2 + f*CWS*LCT
        \note CWS = supply (leaving) chilled water temperature(C)
        \note LCT = leaving condenser fluid temperature(C)
-  A4 , \field Electric Input to Cooling Output Ratio Function of Part Load Ratio Curve Name
+  A4, \field Electric Input to Cooling Output Ratio Function of Part Load Ratio Curve Type
+       \note Two part-load curves are available. 
+       \note The previous one uses the leaving condenser water temperature.
+       \note The new one uses the normalized lift. Lift is the temperature difference between the
+       \note leaving condenser water temperature and the leaving evaporator water temperature. 
+       \note Two new terms are included in the new curve to improve model accuracy when the leaving chilled 
+       \note water temperature deviates from the design conditions especially at high part-load conditions.
+       \required-field
+       \type choice
+       \key UseLeavingCondenserWaterTemperature
```

Comment:
@rongpengzhang Subtle, but I would prefer to drop "Use" from these keywords, LeavingCondenserWaterTemperature or Lift